webview/src/plots/components/App.test.tsx

```diff
@@ -1849,6 +1450,497 @@ describe('App', () => {
+
+      const sections = screen.getAllByTestId(/^plots-section_/)
+      expect(sections.map(section => section.id)).toStrictEqual([
+        'template-multi_0',
+        'template-single_1'
+      ])
+    })
+
+    it('should show a drop target at the end of the section when moving a plot from one section to another but not over any other plot', async () => {
+      renderAppWithOptionalDataInDragAndDropMode({
+        template: complexTemplatePlotsFixture
+      })
+
+      const bottomSection = screen.getByTestId(NewSectionBlock.BOTTOM)
+      const aSingleViewPlot = screen.getByTestId(join('plot_other', 'plot.tsv'))
+
+      dragAndDrop(aSingleViewPlot, bottomSection)
+
+      await screen.findByTestId('plots-section_template-single_2')
+      const anotherSingleViewPlot = screen.getByTestId(
+        join('plot_logs', 'loss.tsv')
+      )
+
+      dragEnter(
+        anotherSingleViewPlot,
+        'template-single_0',
+        DragEnterDirection.LEFT
+      )
+
+      expect(screen.getByTestId('plot_drop-target')).toBeInTheDocument()
+    })
+
+    it('should show a drop target at the end of the template plots section when moving a plot inside of one section but not over any other plot', () => {
+      renderAppWithOptionalDataInDragAndDropMode({
+        template: complexTemplatePlotsFixture
+      })
+
+      const aSingleViewPlot = screen.getByTestId(join('plot_other', 'plot.tsv'))
+
+      dragEnter(aSingleViewPlot, 'template-single_0', DragEnterDirection.LEFT)
+
+      expect(screen.getByTestId('plot_drop-target')).toBeInTheDocument()
+    })
+
+    it('should drop a plot at the end of the template plots section when moving a plot inside of one section but not over any other plot', () => {
+      renderAppWithOptionalDataInDragAndDropMode({
+        template: complexTemplatePlotsFixture
+      })
+
+      const aSingleViewPlot = screen.getByTestId(join('plot_other', 'plot.tsv'))
+      const topSection = screen.getByTestId('plots-section_template-single_0')
+
+      dragAndDrop(aSingleViewPlot, topSection)
+      const plots = within(topSection).getAllByTestId(/^plot_/)
+
+      expect(plots.map(plot => plot.id)).toStrictEqual([
+        join('logs', 'loss.tsv'),
+        join('other', 'plot.tsv')
+      ])
+    })
+
+    it('should show a drop target at the end of the custom plots when moving a plot inside the section but not over any other plot', () => {
+      renderAppWithOptionalDataInDragAndDropMode(
+        {
+          custom: customPlotsFixture,
+          template: templatePlotsFixture
+        },
+        true
+      )
+
+      const plots = screen.getAllByTestId(/summary\.json/)
+
+      dragEnter(plots[0], 'custom-plots', DragEnterDirection.LEFT)
+
+      expect(screen.getByTestId('plot_drop-target')).toBeInTheDocument()
+    })
+
+    it('should show a drop a plot at the end of the custom plots when moving a plot inside the section but not over any other plot', () => {
```

Comment:
[Q] `should show a drop target`?

Reply:
or zone to be consistent with below

Reply:
Moved these tests and didn't notice this. I'll update.